src/Tools/Source/RunTests/ProcDumpUtil.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+
+namespace RunTests
+{
+    internal struct ProcDumpInfo
+    {
+        private const string s_KeyProcDumpFilePath = "ProcDumpFilePath";
+        private const string s_KeyDumpDirectory = "ProcDumpOutputPath";
```

Comment:
> Otherwise it reads that it's a static variable that is the directory to dump from.

I have the opposite problem. When they have the same name as the environment variable it reads like a value, not a key. That is why I prefix them with Key to make their intent clearer.